frontend/src/bundles/common/pages/base.tsx

```diff
@@ -1,7 +1,161 @@
 import React from 'react';
 
+import { Button } from '../components/components';
+// import { ButtonSize, ButtonVariant } from '../enums/enums';
```

Comment:
```suggestion
```

Reply:
Removed in [a14e6a4](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/21/commits/a14e6a4d4d44118fcf3c291f456d982a528d7a7b)